datahub-web-react/src/app/search/sidebar/useBrowsePagination.tsx

```diff
@@ -30,8 +31,9 @@ const useBrowsePagination = ({ skip }: Props) => {
     const total = latestData?.browseV2?.total ?? -1;
     const done = !!latestData && groups.length >= total;
 
-    const [getBrowseResultsV2, { data, error, refetch }] = useGetBrowseResultsV2LazyQuery({
+    const [getBrowseResultsV2, { data, error, loading, refetch }] = useGetBrowseResultsV2LazyQuery({
         fetchPolicy: 'cache-first',
+        notifyOnNetworkStatusChange: true,
```

Comment:
ah so we need this for `loading` to be up to date right? and that will cause some more re-renders? i'm not too worried about that though.

Reply:
Yeah it'll only swap to loading if we're loading a new page though

Reply:
Probably don't need the loading check with the ref addition, removing.